Modules/socketmodule.c

```diff
@@ -5103,14 +5115,25 @@ static int sock_cloexec_works = -1;
 #endif
 
 /*ARGSUSED*/
+
+/*[clinic input]
+_socket.socket.__init__
```

Comment:
You can reduce the amount of lines changed with this trick:

```suggestion
_socket.socket.__init__ as sock_initobj
```